config/motoros2_config.yaml

```diff
@@ -284,3 +284,35 @@ publisher_qos:
 #
 # DEFAULT: false
 #ignore_missing_calib_data: false
+
+#-----------------------------------------------------------------------------
+# Should MotoROS2 broadcast debug messages?
+#
+# If enabled, this will broadcast log messages on the network on port UDP 21789.
+# the user can use the debug script provided in the distribution of MotoROS2 to
+# monitor the state of the robot, identify problems, and debug their code.
+#
+# The debug script is available under the Yaskawa-Global/motoros2 repository in
+# the tools directory
+# https://github.com/Yaskawa-Global/motoros2/tree/main/tools
+#
+# DEFAULT: true
+#userlan_debug_broadcast_enabled: true
```

Comment:
I might have a slight preference for removing the `userlan_` prefix here.

From a user perspective, there are no other LAN ports. The distinction between the port used for the PP and the 'other one(s)' is only made by the OS. Even from M+ we can't really use any other ports.

It would also avoid the (implied) relation between the LAN monitoring and the debug broadcast -- which doesn't exist.

Reply:
Should it be removed from the `userlan_monitoring_` config options as well, or just the `userlan_debug_broadcast_` options?

Reply:
I'd consider everything not related to the new functionality you're introducing as out-of-scope for this PR.